startup_gateway/chat/tests/test_websocket.py

```
@@ -32,88 +32,284 @@ def setUpClass(cls):
+
+        await asyncio.sleep(0.1)
+
+        await comm1.send_json_to({
+            'type': 'send_message',
+            'conversation_id': conversation_id,
+            'body': 'Test delivery'
+        })
+
+        await asyncio.sleep(0.2)
+
+        msg1 = await comm1.receive_json_from(timeout=5)
```

Ruff (0.15.2)

[error] Local variable `msg1` is assigned to but never used

Remove assignment to unused variable `msg1`

(F841)